packages/api-tests/test/window.spec.js

```diff
@@ -112,7 +114,7 @@ describe('Window API', function(done) {
 
     });
 
-    it('Should get the bounds of the window #ssf.Window.getBounds', function() {
+    it('Should get the bounds of the window @no-browser #ssf.Window.getBounds', function() {
```

Comment:
Adding the window core methods into a separate file sounds like a better idea. It will also reduce the size of the test file we have at the moment too.